### PR TITLE
Add a default expiration to the JWT

### PR DESCRIPTION
**Pull request by dunglas**
Integration of https://github.com/symfony/mercure/pull/52.

### CODE REVIEW

**Review by chalasr — APPROVED** [review]
Probably worth a CHANGELOG entry

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -86,6 +87,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->end()
                     ->end()
                     ->scalarNode('default_hub')->end()
+                    ->integerNode('default_cookie_lifetime')->defaultNull()->info('Default lifetime of the cookie containing the JWT, in seconds. Defaults to the value of "framework.cookie_lifetime".')->end()
```
Comment (nicolas-grekas) [inline src/DependencyInjection/Configuration.php:90]: framework.session.cookie_lifetime

File: src/DependencyInjection/MercureExtension.php [hunk 2]
```diff
@@ -251,6 +252,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->register(Authorization::class)
             ->addArgument(new Reference(HubRegistry::class))
+            ->addArgument($config['default_lifetime'] ?? $container->get('session.storage.options', ContainerInterface::NULL_ON_INVALID_REFERENCE)['cookie_lifetime'] ?? null)
```
Comment (chalasr) [inline src/DependencyInjection/MercureExtension.php:255]: `session.storage.options` is not a service but a parameter,  right?

Comment (dunglas) [reply src/DependencyInjection/MercureExtension.php]: Yes my bad. Fixed now using a compiler pass and the proper call to `getParameter`

**Pull request merged**